### DISCUSSION TITLE
ldap login not working - login only via mail

### DISCUSSION THREAD

**Discussion by Alsterdetektive1** [firefly-iii/firefly-iii#5942] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I tried to configure LDAP login for my firefly 3 instance (installed via docker-compose). After configuring that it was not working. "User does not exist" from the login page. I followed the whole instructions on https://docs.firefly-iii.org/firefly-iii/advanced-installation/authentication/
Manually runned in the  container the "composer" commands and used the example ldap settings with "einstein" login. But when I tried with the given credentials it was not working with the error message "enter a valid email-adress". 
So what to do, neither my ldap nor yours are working..

### Expected behaviour

Login via openLdap shall work

### Debug information

enter a valid email-adress"

### Steps to reproduce

1. Install via docker-compose
2. Run compose commands in docker container
3. Configure LDAP Settings like in example page
4. try to login

**Comment by JC5** [comment]
I'm dropping LDAP support for the next release. I would configure something like Authelia.
